draft-ietf-quic-recovery.md

```diff
@@ -720,9 +720,14 @@ Pseudocode for OnAckReceived and UpdateRtt follow:
     // ack-eliciting, update the RTT.
     if (sent_packets[ack.largest_acked] &&
         sent_packets[ack.largest_acked].ack_eliciting):
+      largest_acked_packet = ack.largest_acked
```

Comment:
This doesn't work because the largest_acked in the ACK might be newly acknowledged, but not the largest ever acknowledged.

```suggestion
      if ack.largest_acked > largest_acked_packet:
         largest_acked_packet = ack.largest_acked
```